app/helpers/patient_query_helper.rb

```diff
@@ -472,8 +472,15 @@ def advanced_filter_relative_date(patients, field, filter, tz_diff, type)
       timespan = filter[:value][:number].to_i.months if filter[:value][:unit] == 'months'
       return patients if timespan.nil?
 
-      timeframe = { after: (timespan.ago - tz_diff).beginning_of_day, before: local_current_time } if filter[:value][:when] == 'past'
-      timeframe = { after: local_current_time, before: (timespan.from_now - tz_diff).end_of_day } if filter[:value][:when] == 'next'
+      case filter[:value][:operator]
+      when 'less-than'
+        timeframe = { after: (timespan.ago - tz_diff).beginning_of_day, before: local_current_time } if filter[:value][:when] == 'past'
+        timeframe = { after: local_current_time, before: (timespan.from_now - tz_diff).end_of_day } if filter[:value][:when] == 'future'
+      when 'more-than'
+        timespan += 1.day if filter[:filterOption][:hasTimestamp] == false
+        timeframe = { before: timespan.ago - tz_diff } if filter[:value][:when] == 'past'
```

Comment:
I'm not sure if this is correct, but I also haven't figured out what it should be yet

Reply:
This behaves correctly based on the requirements the PO team gave me are...
- For date fields with a time stamp, go back x units and include any monitorees that have a date before the current time on that day 
- For date fields without a time stamp, go back x units and don't include that day but include all days before

Example: more than 3 days ago  (on today 2/19) should return
- Dates on 2/16 before the current time for date fields with a timestamp
- Dates before 2/16 (2/15 and before) for date fields without a timestamp

Reply:
If it's not behaving that way, let me know